examples/taxis/README.md

```diff
@@ -2,17 +2,41 @@
 
 Simulating real traffic using the [*New York City Taxi Trip Duration* dataset](https://www.kaggle.com/c/nyc-taxi-trip-duration).
 
+## General idea
+
+This proccedure helps understand how beaver can be used to predict trip durations of taxis using an event dataset consisting of pickup date-times, pickup coordinates, arrival coordinates, etc.
+
+* The procedure here is to create an empty model using the river library
+* Serialize the model using dill
+* Upload the model using the beaver http client --> you could also do this using the HTTP API directly (see localhost:3000/docs for more details)
+* Then start receiving trip data in a simulation and make predictions for trip duration
+
+During simulation you should see the error decreasing with time, which can be visualized in the output messages as in:
+
+```
+... a taxi starts a trip with id 000014
+#0000014 departs at 2016-01-01 00:04:57
+... times goes by
+... eventually the trip ends
+#0000014 arrives at 2016-01-01 00:08:37, took 0:03:40, predicted 0:08:41
+```
+
+so for the trip 000014 we can see an error of more then 200% in this case
+
 ## Setup
 
 First of all you need a running instance of Beaver. For instance, you can to the root of this repo and start an instance in the background:
 
 ```sh
-(cd ... && docker-compose up -d)
+(cd ... && docker-compose up -d && docker exec -it app /bin/bash)
```

Comment:
Actually the idea is to run the simulation from the host, not from the instance where Beaver is located.

Reply:
I get what you mean. How about using a virtual environment? A dedicated Docker instance could work too I guess. You decide! But in any case I would rather separate the server from the client.

Reply:
Sorry forgot to answer about that. Actually the train.py script was meant to be used when the only way to update a model was manually. Now there's an option to update it every time a label arrives. You can delete it :)

Reply:
but again, just to undestand, will it be idempotent right? i mean in supervised learning the labels and features would be the same if train.py is executed twice. 
i guess that would not hold true for unsupervised learning (anomalies detection). but this is just a supposition here